includes/utilities.inc

```diff
@@ -115,6 +115,30 @@ function islandora_rest_accessible($endpoint, $request_method, $resource = FALSE
 }
 
 /**
+ * Retrieve REST permissions mapped to core.
+ *
+ * @return array
+ *   A key value pair of this modules permissions,
+ *   mapped to core islandora permissions.
+ */
+function _islandora_rest_permission_map() {
+  return array(
+    ISLANDORA_REST_OBJECT_GET_PERM           => ISLANDORA_VIEW_OBJECTS,
+    ISLANDORA_REST_OBJECT_PUT_PERM           => ISLANDORA_MANAGE_PROPERTIES,
+    ISLANDORA_REST_OBJECT_POST_PERM          => ISLANDORA_INGEST,
+    ISLANDORA_REST_OBJECT_DELETE_PERM        => ISLANDORA_PURGE,
+    ISLANDORA_REST_DATASTREAM_GET_PERM       => ISLANDORA_VIEW_OBJECTS,
+    ISLANDORA_REST_DATASTREAM_PUT_PERM       => ISLANDORA_ADD_DS,
+    ISLANDORA_REST_DATASTREAM_POST_PERM      => ISLANDORA_METADATA_EDIT,
```

Comment:
I think the datastream `PUT` and `POST` are backwards... The [permission titles](https://github.com/discoverygarden/islandora_rest/blob/7.x/islandora_rest.module#L25-L26) look like it should be `POST => ISLANDORA_ADD_DS`, and `PUT => ISLANDORA_METADATA_EDIT`.